backend/src/main/java/me/performancereservation/domain/admin/AdminService.java

```diff
@@ -0,0 +1,25 @@
+package me.performancereservation.domain.admin;
+
+import lombok.RequiredArgsConstructor;
+import me.performancereservation.global.exception.AppException;
+import me.performancereservation.global.exception.ErrorCode;
+import me.performancereservation.global.exception.ErrorType;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class AdminService implements UserDetailsService {
+
+    private final AdminRepository adminRepository;
+
+    @Override
+    public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException {
+        Admin admin = adminRepository.findById(username)
+                .orElseThrow(() -> new AppException(ErrorCode.ADMIN_NOT_FOUND, ErrorType.SERVICE));
+
```

Comment:
```suggestion
                .orElseThrow(() ->ErrorCode.ADMIN_NOT_FOUND.domainException("예외 내용");

```

형태로 작성하면 알아서 예외 객체가 생성됩니다!

Reply:
아 감사합니다! 수정해보겠습니다!